openapi/components/schemas/RiskScoreAllowListRecord.yaml

```
@@ -0,0 +1,33 @@
+type: object
+description: Risk score allow list record.
+required:
+  - type
+  - value
+properties:
+  id:
+    type: string
+    description: ID of the allow list record.
+    readOnly: true
+    maxLength: 50
+    example: rsal_0YVF9605RKC62BP14NE2R7V2XT
+  type:
+    type: string
+    description: Type of exclusion from adding to blocklist.
+    enum:
+      - address
+      - bank-account
+      - bin
+      - country
+      - customer-id
+      - email
+      - email-domain
+      - fingerprint
+      - ip-address
+      - payment-card
+  value:
+    type: string
+    description: Value of allowed record.
```

```suggestion
    description: Value of the allow list record.
```
So it matches `id` property description. Not an expert here thou.

---

Updated, thanks @lexerom